### PR TITLE
adding ci to make latest helm release 

### PR DESCRIPTION
**Pull request by Shreyas220**

### CODE REVIEW

**Comment by daemon1024** [comment]
Ref https://github.com/kubearmor/kubearmor/issues/1377

File: .github/workflows/helm-release.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+      - uses: azure/setup-helm@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Update Chart.yaml
+        run: |
+          STABLE_VERSION=`cat STABLE-RELEASE`
+          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" helm/sidekick/Chart.yaml
+          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" helm/sidekick/Chart.yaml
```
Comment (DelusionalOptimist) [inline .github/workflows/helm-release.yml]: Will we be releasing a stable version of sidekick? If yes, then let's also add the STABLE_RELEASE file.
But then we'll also need to think about publishing stable docker images.

File: .github/workflows/helm-release.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+      - name: Publish Helm chart
+        uses: stefanprodan/helm-gh-pages@master
+        with:
+          # Access token which can push to a different repo in the same org
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
```
Comment (DelusionalOptimist) [inline .github/workflows/helm-release.yml:33]: This secret needs to be created.
cc @daemon1024

File: .github/workflows/helm-release.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          charts_dir: helm/
+          # repo where charts would be published
+          owner: kubearmor
+          repository: charts
+          branch: gh-pages
+          charts_url: https://sidekick.github.io/charts
```
Comment (DelusionalOptimist) [inline .github/workflows/helm-release.yml]:
```suggestion
          charts_url: https://kubearmor.github.io/charts
```

**Review by DelusionalOptimist — CHANGES_REQUESTED** [review]

**Review by DelusionalOptimist — APPROVED** [review]

**Pull request merged**